### DISCUSSION TITLE
When the server is not accessible, can plugins be loaded without server through audio gridder?

### DISCUSSION THREAD

**Discussion by petetwopointoh** [apohl79/audiogridder#411] [Q&A]
Hi Andreas, my question is this - if i load up an instance of audio gridder, and can’t access my server, is there a way that I can easily just have audiogridder pull up the plug-in on the computer that I’m using, with the same settings. This is my main issue right now with using audiogridder. I use my laptop and travel sometimes, so when I’m away from my server, unfortunately I cannot use audiogridder, and any work that i have done is not accessible. I have a stable version of audiogridder so I haven’t updated in a while. Would be nice if I could just switch back and forth. Thank you!

**Comment by apohl79** [comment]
You can run the server on your laptop and connect the plugin to the local server. It will keep the plugin settings, as all remote plugin settings are stored in the DAW session.

I know this is cumbersome for large sessions with many AG plugins. I will improve the way you can work with AG locally. And there should also be a way to change the server for a group of plugins.

**Reply by petetwopointoh** [reply]
Good to know, thank you!!! I’ll check that out and start messing with it. Would be great to be able to change from audiogridder to a native plug-in, but maybe down the line this can be somehow implemented (a carbon or soundgridder concept). Again, thank you Andreas for this great plug-in, and look forward to what else you have in store for us! Very much appreciated 🤘

**Reply by apohl79** [reply]
One note though, what I've seen is that some vendors would use different VST plugin ID's for the same plugin on macOS vs Windows. So it is likely to work better when you run the same OS on the server and workstation.
